src/views/DashboardPage/DashboardPage.scss

```diff
@@ -24,6 +25,6 @@
 
     .bottom {
         display: flex;
-        height: 100%;
+        height: 90%;
```

Comment:
just a question, why 90%?

Reply:
it fixes the problem, the problem was that it is inside flexed conatiner (App) but app includes also the top bar so I did the topbar 10% and the page 90%